circus-train-integration-tests/src/test/data/com/hotels/bdp/circustrain/integration/CircusTrainHdfsHdfsIntegrationTest/unpartitioned-single-table-avro-schema.yml

```diff
@@ -6,5 +6,9 @@ table-replications:
     replica-table:
       table-name: ct_table_u_managed_copy
       table-location: ${circus-train-runner.replica-warehouse-uri}${circus-train-runner.database-name}/ct_table_u_managed_copy
-avro-serde-options:
-  base-url: ${circus-train-runner.replica-warehouse-uri}${circus-train-runner.database-name}/
\ No newline at end of file
+transform-options:
```

Comment:
Instead of re-using this file, just create a new one which has your multiple transformations in it.

Reply:
Separated, and added a new IT for transformation overrides